fractal_tasks_core/lib_regions_of_interest.py

```diff
@@ -372,6 +361,35 @@ def is_ROI_table_valid(*, table_path: str, use_masks: bool) -> Optional[bool]:
         return False
 
 
+def are_ROI_table_columns_valid(*, table: ad.AnnData) -> Optional[bool]:
+    """
+    Verify some validity assumptions on a ROI table.
+
+    This function reflects our current working assumptions (e.g. the presence
+    of some specific columns); this may change in future versions.
+
+    Args:
+        table: AnnData table to be checked
+
+    Returns:
+        Always `None` if `use_masks=False`, otherwise return whether the table
+            is valid for masked loading.
+    """
```

Comment:
This function always returns `None`, thus I guess that its return type is `None` and its docstring entry
```
    Returns:
        Always `None` if `use_masks=False`, otherwise return whether the table
            is valid for masked loading.
```
is not meant to be there.